### ISSUE TITLE
Feature request: disable checking for updates

### ISSUE THREAD

**Issue by LankyCyril** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1019]
**OS:** Windows

**Issue Descriptions:** The "the new version is here" window is _**very persistent**_, and there is no entry in "Desktop settings" to disable it. I want to update whenever I feel I should, not whenever there is a new version.

**Steps to Reproduce:** Download and install the version before last, run it and wait a little bit.

**Comment by jostrander** [comment]
I'm on a platform without auto update,  however I believe this was fixed in #966. Does the description there fit what's happening for you?

**Comment by LankyCyril** [comment]
I'm sorry that I forgot to specify that yes, it isn't 12 seconds (anymore), it's 20 minutes. But it's still very persistent in my book. It would be great to have an option of how often to prompt for updates that **includes "never"**.

**Comment by MarshallOfSound** [comment]
@LankyCyril Unfortunately normally the updates for this application include fixes for BREAKING bugs caused by Google changing something.  This means that until you update your experience is not going to be the best hence the update notifications.  I don't think personally 20 minutes is too annoying and if you simply click "update" the app will relaunch back to where you left it in a few seconds.

TLDR: Updates aren't just features in this application, they fix BREAKING bugs caused by Google that NEED to be fixed or the player stops working

**Comment by LankyCyril** [comment]
Great point, thank you. I hope you don't mind that I added your explanation to the FAQ.

**Comment by MarshallOfSound** [comment]
@LankyCyril Not at all, thanks for adding it :+1:  :smile:
